packages/react-ui-core/src/Counter/Counter.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import classNames from 'classnames'
+
+export default class Counter extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      count: this.props.count,
+    }
+    this.handleClick = this.handleClick.bind(this)
+  }
+  handleClick(count) {
+    this.props.handleClick(count)
+    this.setState({
+      count,
+    })
+  }
+  render() {
+    const { count } = this.state
+    const { theme, label, decrementUnit, incrementUnit } = this.props
+    return (
+      <div className={theme.Counter}>
+        <div className={theme['Counter-label']}>
+          {label}
+        </div>
+        <div className={theme['Counter-label-active']}>
+          <span
+            onClick={() => this.handleClick(this.state.count + 1)}
+            className={classNames('incrementUnit', theme['Counter-content'], theme['Counter-shape'])}
```

Comment:
because we use locally scoped css modules we should not hard code classes in any of the components. lets find a better way to target this item here and in your tests